graylog2-web-interface/src/views/components/widgets/MessageTable.jsx

```diff
@@ -220,6 +229,12 @@ class MessageTable extends React.Component<Props, State> {
                     <Field type={this._fieldTypeFor(selectedFieldName, fields)}
                            name={selectedFieldName}
                            queryId={activeQueryId} />
+                    {editing && (
```

Comment:
I guess we need to wrap that in an `<IfInteractive>...</IfInteractive>` block?

Reply:
We are using `<IfInteractive>` only to differentiate between the big display mode. I could create a wrapper like `<IfEditing>` but we are using the edit check just a few times.